cumulusci.yml

```diff
@@ -926,25 +912,6 @@ flows:
                 options:
                     class_name: RD2_DataMigration_BATCH
 
-    enable_rd2_managed:
```

Comment:
Do we know if any downstream or Robot automation is still using this flow?

Reply:
Great question, and I think when I make these changes initially I didn't realize that Robot could call a flow this way. The `npsp_robot` file does explicitly call this. If _managed_ flows are no longer needed, I'd prefer to fix the robot file accordingly. However, that [robot file actually goes a lot deeper](https://github.com/SalesforceFoundation/NPSP/blob/master/robot/Cumulus/resources/NPSP.robot#L431). If unmanaged, it looks like it's recreated the entire `enable_rd` flow in robot. I'm not really sure why that is the case though. I should just be able to call the one flow regardless of managed or unmanaged. 
@rjanjanam What do you think?

Reply:
However, that may be more than I want to take on in this PR. I think I'll revert this one change and we can address the use of `enable_rd2` vs. `enabled_rd2_managed` separately.

Reply:
That is great if one flow can serve for both. I am address in my next PR if you want to make that change in NPSP.robot. Currently I am checking the namespace and the backend call to ensure RD2 is enabled or not and running the flow accordingly. So I am needing both. I had to implement the entire blow for QA back then as I did not know I can call it from Run process.
```    [Documentation]           Checks if Rd2 settings are already enabled and then run the scripts to enable RD2
    ${is_rd2_enabled} =       API Query Recurrring Donation Settings For RD2 Enablement
    ${ns} =                   Get Npsp Namespace Prefix
    Log To Console             ${ns}
    Run Keyword if            "${is_rd2_enabled}"!="True" and "${ns}"!="npsp__"
    ...                       Enable RD2QA
    Run Keyword if            "${is_rd2_enabled}"!="True" and "${ns}"=="npsp__"
    ...                       Run Process     cci  flow  run  enable_rd2_managed

Reply:
Thank @rjanjanam. The one flow does work fine, at least when running from the command line. However, before I remove the _managed_ flow from the yml, I'd feel more comfortable knowing that you've updated `npsp.robot` to confirm that just calling the `enable_rd2` directly works fine for both managed and unmanaged in a robot context.